src/components/Pattern/Pattern.jsx

```diff
@@ -0,0 +1,228 @@
+            <path d="M562.7 27.27C560.046 27.2786 557.47 28.1666 555.375 29.7951C553.28 31.4235 551.783 33.7006 551.12 36.27L516.66 38.84L512.33 0H503.16L507.16 41.17L486 46L497.13 73.82C495.498 75.4769 494.49 77.6481 494.278 79.9642C494.067 82.2803 494.664 84.5982 495.969 86.5235C497.274 88.4488 499.205 89.8626 501.435 90.5243C503.664 91.1859 506.054 91.0546 508.198 90.1526C510.342 89.2506 512.107 87.6337 513.193 85.577C514.279 83.5203 514.618 81.1508 514.154 78.8719C513.69 76.5929 512.45 74.5452 510.647 73.0772C508.843 71.6092 506.586 70.8115 504.26 70.82C504.07 70.82 503.89 70.82 503.7 70.82L503.84 49.43L552.33 45.09C553.359 46.9443 554.86 48.4929 556.682 49.5783C558.503 50.6636 560.58 51.2469 562.7 51.2687C564.821 51.2906 566.909 50.7502 568.752 49.7026C570.596 48.655 572.129 47.1377 573.195 45.305C574.262 43.4723 574.824 41.3897 574.824 39.2693C574.824 37.1488 574.262 35.0663 573.195 33.2336C572.129 31.4009 570.596 29.8836 568.752 28.836C566.909 27.7885 564.82 27.2481 562.7 27.27ZM508.11 79.11C508.475 79.8699 508.607 80.7213 508.489 81.5562C508.37 82.3911 508.006 83.1719 507.443 83.7999C506.88 84.4278 506.144 84.8746 505.327 85.0835C504.51 85.2925 503.649 85.2544 502.854 84.9739C502.059 84.6933 501.365 84.1831 500.859 83.5078C500.354 82.8325 500.061 82.0225 500.017 81.1804C499.973 80.3382 500.179 79.5019 500.611 78.7773C501.042 78.0527 501.679 77.4724 502.44 77.11C503.457 76.6257 504.625 76.5642 505.688 76.939C506.751 77.3139 507.622 78.0945 508.11 79.11ZM564.86 43.82C563.952 44.2474 562.939 44.3959 561.947 44.2469C560.955 44.0979 560.029 43.658 559.287 42.9828C558.546 42.3077 558.021 41.4277 557.779 40.4541C557.538 39.4806 557.59 38.4572 557.93 37.5136C558.271 36.57 558.883 35.7485 559.69 35.1531C560.498 34.5576 561.463 34.215 562.465 34.1686C563.467 34.1222 564.461 34.374 565.319 34.8923C566.178 35.4105 566.864 36.1719 567.29 37.08C567.858 38.2966 567.921 39.6884 567.466 40.9513C567.01 42.2143 566.073 43.2456 564.86 43.82Z" fill="#F2F2F2"/>
+          </g>
+          <defs>
+            <clipPath id="clip0">
+              <rect width="1377" height="300" fill="white"/>
+            </clipPath>
+          </defs>
+        </svg>
+      `
+      };
+      break;
+    default:
+  }
+
+  return (
+    <PatternWrapper>
```

Comment:
Pass `color` straight in to `PatternWrapper`.

Like so: `<PatternWrapper color={color}>`